feature/experimental/policy/policy_vrf_selection/ate_tests/protocol_dscp_rules_for_vrf_selection_test/protocol_dscp_rules_for_vrf_selection_test.go

```diff
@@ -472,7 +472,10 @@ func TestPBR(t *testing.T) {
 			pfpath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).PolicyForwarding()
 
 			//configure pbr policy-forwarding
+			dutConfNIPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance)
+			gnmi.Replace(t, dut, dutConfNIPath.Type().Config(), oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
 			gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).PolicyForwarding().Config(), tc.policy)
+			
```

Comment:
I believe what I suggested is "Update" instead of Replace, like the following to replace line 477. I was wondering if the two "Replace" in a row can retain the first configuration?
gnmi.Update(t, dut, dutConfNIPath.PolicyForwarding().Config(), tc.policy)

Reply:
Updated the code with gnmi.Update instead of gnmi.Replace for policy forwarding as suggested